internal/ingress/controller/store/store.go

```diff
@@ -538,12 +538,12 @@ func (s *k8sStore) updateSecretIngressMap(ing *extensions.Ingress) {
 		"auth-tls-secret",
 	}
 	for _, ann := range secretAnnotations {
-		secrName, err := parser.GetStringAnnotation(ann, ing)
+		secrKey, err := objectRefAnnotationNsKey(ann, ing)
 		if err != nil {
+			glog.Errorf("error reading secret reference in annotation %q: %s", ann, err)
```

Comment:
Just realized `parser.checkAnnotation()` throws `errors.ErrMissingAnnotations` if the Ingress object being processed does not have any annotation. This is probably something we want to silence.